### DISCUSSION TITLE
希望添加的一些功能

### DISCUSSION THREAD

**Discussion by 840751087** [Richasy/Bili.Uwp#1090] [Polls]
之前一直用的官方PC版本有个功能我个人觉得不错希望添加一下回车全屏esc返回等一些快捷键

**Comment by Richasy** [comment]
哔哩有自己的 [快捷键](https://github.com/Richasy/Bili.Uwp/wiki/%E5%BF%AB%E6%8D%B7%E9%94%AE)
